src/core/annotation.js

```diff
@@ -1098,6 +1099,18 @@ class MarkupAnnotation extends Annotation {
         this.data.color = null;
       }
     }
+
+    if (dict.has("RC")) {
+      const htmlText = XFAFactory.getRichTextAsHtml(dict.get("RC"));
+      if (
+        htmlText &&
+        (!this.data.contentsObj ||
+          !this.data.contentsObj.str ||
+          this.data.contentsObj.str === htmlText.text)
+      ) {
+        this.data.richText = htmlText.html;
+      }
```

Comment:
Let's improve the API, for a non-viewer consumer, by always returning the `richText` and instead decide on what to use in the display-layer; i.e. let's instead do:
```javascript
      this.data.richText = XFAFactory.getRichTextAsHtml(dict.get("RC"));
```